Casks/a/ape.rb

```diff
@@ -0,0 +1,17 @@
+cask "ape" do
+  version :latest
+  sha256 :no_check  # No versioned URL, so we skip checksum verification
```

Comment:
```suggestion
  sha256 :no_check
```

We don't need the comment here because the lack of a versioned url makes this self-documenting.